queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/aliases.py

```diff
@@ -31,7 +31,7 @@ def update_aliases(business: Business, aliases: Dict) -> Dict:
     if modified_aliases := aliases.get('modified'):
         for current_alias in business.aliases.all():
             for mod_alias in modified_aliases:
-                if current_alias.alias == mod_alias.get('oldValue'):
+                if current_alias.alias == str(mod_alias.get('oldValue')).upper():
```

Comment:
Can you add the .upper to both sides of this comparison? Not %100 sure the data will always be consistent when we load corps from COLIN so we might have some cases where current_alias.alias is mixed case

Reply:
Sure. I did not add based on the implementation below for "new" aliases